src/content/docs/browser/browser-monitoring/browser-pro-features/session-replay.mdx

```diff
@@ -267,6 +274,15 @@ In the UI, you can find session replays in two places:
 
 </Steps>
 
+## (Optional) Manually record session replays [#manual-replays]
+
+You can manually invoke a session to be collected by using these two API methods:
+
+* [`NREM.startSessionReplay`](/docs/browser/new-relic-browser/browser-apis/start/)
+* [`NREM.stopSessionReplay`](/docs/browser/new-relic-browser/browser-apis/stop/)
+
+If you use these methods, you'll need to set both sampling rates in application settings to 0%. This is beneficial if you’d like a user to opt in to having user interactions recorded for page performance and observability purposes.  
+
```

Comment:
@metal-messiah @CherylFrankenfield I think this section needs more info, so here are some questions I have:

1. When it says "both sampling rates", should this just say "all sampling rates" to include the new session traces one?
2. The sentence is about "This is beneficial if you'd like a user to opt in to having interactions recorded" is confusing. What does this mean? Why would a manual recording be any different than a standard sampling,  AKA what's the use case for using this manual process?
3. How exactly does this work? As a reader, if I follow the docs links out to the API start/stop docs, it's not really helpful/doesn't tell me how it's applicable for session replay, so I think any relevant content from the other docs should be implemented/contextualized here.